flask_security/core.py

```diff
@@ -121,81 +127,80 @@
 #: Default Flask-Security messages
 _default_messages = {
     'UNAUTHORIZED': (
-        'You do not have permission to view this resource.', 'error'),
+        _('You do not have permission to view this resource.'), 'error'),
     'CONFIRM_REGISTRATION': (
-        'Thank you. Confirmation instructions have been sent to %(email)s.',
+        _('Thank you. Confirmation instructions '
+          'have been sent to %(email)s.'),
         'success'),
     'EMAIL_CONFIRMED': (
-        'Thank you. Your email has been confirmed.', 'success'),
+        _('Thank you. Your email has been confirmed.'), 'success'),
     'ALREADY_CONFIRMED': (
-        'Your email has already been confirmed.', 'info'),
+        _('Your email has already been confirmed.'), 'info'),
     'INVALID_CONFIRMATION_TOKEN': (
-        'Invalid confirmation token.', 'error'),
+        _('Invalid confirmation token.'), 'error'),
     'EMAIL_ALREADY_ASSOCIATED': (
-        '%(email)s is already associated with an account.', 'error'),
+        _('%(email)s is already associated with an account.'), 'error'),
     'PASSWORD_MISMATCH': (
```

Comment:
I have found it in an old commit: https://github.com/jirikuncar/flask-security/blob/f3014d01dfd356481e43c947e24c56ee2669a05c/flask_security/forms.py#L130-L133